src/plugins/console/public/application/components/output_panel_empty_state.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent } from 'react';
+import {
+  EuiHorizontalRule,
+  EuiSpacer,
+  EuiTitle,
+  EuiLink,
+  EuiText,
+  EuiFlexGroup,
+  EuiFlexItem,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useServicesContext } from '../contexts';
+
+export const OutputPanelEmptyState: FunctionComponent = () => {
+  const { docLinks } = useServicesContext();
+
+  return (
+    <EuiFlexGroup
+      alignItems="center"
+      className="conApp__outputPanel__emptyState"
+      data-test-subj="consoleOutputPanelEmptyState"
+    >
+      <EuiFlexItem>
+        <EuiTitle size="s">
```

Comment:
A slight modification: There's been a push to have us make an effort to be more consistent with EUI, so with that in mind, would it be possible to use the [EUI EmptyPrompt](https://eui.elastic.co/#/display/empty-prompt) here? I think it's probably the better component than something custom to match my design. 

What do you think? 🙏

```
<EuiEmptyPrompt
      title={<h2>Enter a new request</h2>}
      body={
        <p>
          When you run a request in the input pane, you
          will see the output response here.
        </p>
      }
      footer={
        <>
          <EuiTitle size="xxs">
            <h3>Want to learn more?</h3>
          </EuiTitle>
          <EuiLink href="#" target="_blank">
            Read the Console docs
          </EuiLink>
        </>
      }
    />
```

Reply:
Sure, here's how it would look with the EUI empty prompt:

<img width="1486" alt="Screenshot 2024-08-20 at 19 29 44" src="https://github.com/user-attachments/assets/8f2ee2c1-8ad0-4224-a75b-b8f4581ae64b">

It looks good to me and I don't have a strong preference between this and the original design. Let me know what you think.

Reply:
Could we also update the screenshots in the PR description?

Reply:
Sure! I haven't pushed these changes yet - wanted to see if Julian has any further suggestions on this. I'll update the screenshots once I add the changes!

Reply:
Added the empty prompt from above with [9b37970](https://github.com/elastic/kibana/pull/190627/commits/9b37970c7c02f58821de6856a2f1ac10fc7fa2fe) and updates the screenshots!